docs/operators/recombine.md

```diff
@@ -11,9 +11,10 @@ The `recombine` operator combines consecutive logs into single logs based on sim
 | `on_error`       | `send`           | The behavior of the operator if it encounters an error. See [on_error](/docs/types/on_error.md). |
 | `is_first_entry` |                  | An [expression](/docs/types/expression.md) that returns true if the entry being processed is the first entry in a multiline series. |
 | `is_last_entry`  |                  | An [expression](/docs/types/expression.md) that returns true if the entry being processed is the last entry in a multiline series. |
-| `combine_field`  | required         | The [field](/docs/types/field.md) from all the entries that will recombined with newlines. |
+| `combine_field`  | required         | The [field](/docs/types/field.md) from all the entries that will recombined. |
+| `combine_with`   | `\n`             | The string that is put between the combined entries. This can be an empty string as well. |
```

Comment:
Nit - since a user should expect different results based on whether or not this value is quoted, I think we need to be precise here. Specifically, the user would have to specify `"\n"` in order to replicate the default behavior, so I think we should show that exact value here.

Reply:
Yes I fully agree, we should be explicit about the types of the values.

Reply:
Typically, it doesn't matter if the user writes `"foo"` or `foo`. When read, the value becomes the same either way.

In this case, it does matter whether the user writes `"\n"` or `\n`. I think it is enough to show the value here that is actually interpreted the same as the default value. 

```suggestion
| `combine_with`   | `"\n"`             | The string that is put between the combined entries. This can be an empty string as well. |
```

Reply:
Fixed in https://github.com/open-telemetry/opentelemetry-log-collection/pull/315/commits/649bfc8a74dc505026d625f3135db78935b220a3.

Also updated examples in the docs, please take a look @djaglowski.